ci/utils/linux-common-runnner.sh

```
@@ -54,11 +54,15 @@ install_module() {
 run_case() {
     export_or_prefix
 
+    openresty -V
+
     ./bin/apisix init
     ./bin/apisix init_etcd
 
+    export PATH=/usr/local/tongsuo/bin:$PATH
+    openssl version
     git submodule update --init --recursive
-    FLUSH_ETCD=1 prove -I../test-nginx/lib -I./ -r t/gm
+    FLUSH_ETCD=1 prove -I../test-nginx/lib -I./ -r t/gm || cat t/servroot/logs/error.log
```

The test fails, but it looks like `||` hides the error.

---

https://github.com/api7/apisix-plugin-gm/pull/3/checks#step:10:64

---

Sorry, I forgot to remove debug directive last night